WcaOnRails/spec/requests/api_persons_spec.rb

```diff
@@ -53,4 +53,16 @@
       expect(json["personal_records"]["333"]["average"]["best"]).to eq 590
     end
   end
+
+  describe "GET #results" do
+    it "renders properly" do
+      get api_v0_person_results_path(person.wca_id)
+      expect(response).to be_successful
+      json = JSON.parse(response.body)
+      if json[0]
```

Comment:
Sorry, but this doesn't actually test anything ^^ There are no competitors for this competition, so this will always be false. What you want to do is create appropriate records in each test case. In this case you can simply add that before before the `it`:
```rb
let!(:result) { FactoryBot.create :result, person: person }
```
Then you can check if the result gets rendered as expected e.g. `expect(json[0]["id"]).to eq result.id`

Reply:
Nit: the `if` check is no longer needed as we expect to have a result here.